src/components/teamCard1_2023/TeamCard1_2023.css

```diff
@@ -0,0 +1,133 @@
+.codeutsava__team-card1_2023 {
+    padding: 50px 20px;
+    background-color: #fff;
+    text-align: center;
+    overflow: hidden;
+    position: relative;
+    border-radius: 20px;
+    min-width: 320px;
+    max-width: 320px;
+  }
+  
+  .codeutsava__team-card1_2023 .codeutsava__team-card1_2023-img{
+    display: inline-block;
+    height: 170px;
+    width: 170px;
+    margin-bottom: 15px;
+    z-index: 1;
+    position: relative;
+  }
+  
+  .codeutsava__team-card1_2023 .codeutsava__team-card1_2023-img::before {
+    content:"";
+    background-image: url(../../assets/images/teamcard1_2023/teamcard_4_web.png);
```

Comment:
Rename the file as 'web.png' and put it in the images folder. Remove the teamcard1_2023 folder.